Graphana and prometheus fixed

Volodimirich opened the pull request.

Volodimirich committed:
Graphana and prometheus fixed

besteady commented on README.md:
```diff
@@ -56,3 +56,19 @@ We found the following fields and data models useful.
+`http://127.0.0.1:9090/targets?search=`. After redirecting you should
+ check availability of `http://mongodb-exporter:9216/metrics`.
+
+For monitoring Graphana was chosen. To use it you should do next steps:
```
```suggestion
As a dashboard Grafana was chosen. To use it you should do next steps:
```

besteady commented on README.md:
```diff
@@ -56,3 +56,19 @@ We found the following fields and data models useful.
+
+For monitoring Graphana was chosen. To use it you should do next steps:
+1. Go to `http://127.0.0.1:3000` (login: _admin_, password: _pass@123_).
+2. In opened window go to Datasources -> Prometheus. In the new window set next url - `http://prometheus:9090`. After 
```
Can we somehow add `prometheus` automatically with config?

besteady commented on datasource.yml:
```diff
@@ -0,0 +1,49 @@
+      tlsClientKey: "..."
+    version: 1
+    # <bool> allow users to edit datasources from the UI.
+    editable: true
```
Mb remove all comments

```suggestion
    editable: true
    
```
New line at end of file.

besteady commented on docker-compose.yaml:
```diff
@@ -75,15 +72,13 @@ services:
+      MONGODB_URI: 'mongodb://mongodb_exporter:s3cr3tpassw0rd@mongo-db:27017/main?sslmode=disable'
+    links:
+      - mongo-db
+      - prometheus
```
```suggestion
      - prometheus
      
```

Newline at end of file

besteady commented on test/scripts/init-mongo.js:
```diff
@@ -10,3 +10,12 @@ db.createUser({
+        { role: "clusterMonitor", db: "admin" },
+        { role: "read", db: "local" }
+    ]
+})
```
```suggestion
})

```

Newline at end of file

Volodimirich committed:
Small fixes

Volodimirich committed:
Fixed merge conflicts:
# Conflicts:
#	docker-compose.yaml
#	test/scripts/init-mongo.js

Change project architecture

Volodimirich committed:
Final fixes
